README.md

```diff
@@ -51,8 +52,9 @@ list of requirements.
 
 If you want to statically link any codec into your local (static) build of
 libavif, building using one of these scripts and then enabling the associated
-`AVIF_LOCAL_*` is a convenient method, but you must make sure to disable
-`BUILD_SHARED_LIBS` in CMake to instruct it to make a static libavif library.
+`AVIF_LOCAL_*` and setting `AVIF_CODEC_*` to `LOCAL` is a convenient method,
```

Comment:
The new text mentions both the old and new ways, connected by "and". I think we just need to mention the new way ("setting `AVIF_CODEC_*` to `LOCAL`"). If we mention both, they should be connected by "or" because one of them is sufficient.

Reply:
Thx, fixed in https://github.com/AOMediaCodec/libavif/pull/1795 I believe